src/Components/Todo/TodoItem.tsx

```diff
@@ -0,0 +1,62 @@
+import classNames from 'classnames';
+import React, { useState } from 'react';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo;
+  onTodoRemove: (todoId: number) => void;
+  isRemovingCompleted: boolean;
+};
+
+export const TodoItem: React.FC<Props> = ({
+  todo,
+  onTodoRemove,
+  isRemovingCompleted,
+}) => {
+  const [isRemoving, setIsRemoving] = useState<boolean>(false);
```

Comment:
why do you need this state here?
Also, TS already knows about this type, bc you put an initial value
```suggestion
  const [isRemoving, setIsRemoving] = useState(false);
```

Reply:
it is responsible for showing the loader when deleted one todo